src/components/subject-number/SubjectNumber.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { FC } from "react";
+import { Manifestation } from "../../core/utils/types/entities";
+
+export interface SubjectNumberProps {
+  className?: string;
+  shelfmark: NonNullable<Manifestation["shelfmark"]>;
```

Comment:
I find it hard to decide when to use the underlying primitive type (in this case `string`?) and when to use the type of the property of a larger type 🤔 .

`NonNullable<Manifestation["shelfmark"]>` does not seem very readable compared to e.g. `string`. In this case we seem to assume when rendering the component that the value can be stringified anyway. 

Perhaps this is something we can talk about at some point.